app/controllers/asset_manager_redirect_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AssetManagerRedirectController < ApplicationController
+  def show
```

Comment:
I think it might be worth adding something about the possibility of an infinite redirect loop in the commit note for "PublicUploadsController only needs to redirect". Once this change has been deployed, and until we remove the [`/government/uploads` catch-all route from the assets-origin nginx config][1], it's going to be possible to request a made-up asset path (e.g. https://assets-origin.integration.publishing.service.gov.uk/government/uploads/foo/bar/baz.png) and have it result in an infinite redirect.

[1]: https://github.com/alphagov/govuk-puppet/blob/7e6954cda9f7976d552faa12b0019eb054d4772b/hieradata/common.yaml#L1643